tests/auto/schema/schema_objectclass.c

```diff
@@ -30,7 +30,7 @@ Ensure(returns_true_on_successful_append_to_full_capacity_to_objectclasses) {
     TALLOC_CTX *ctx = talloc_new(NULL);
 
     struct ldap_schema_t *schema = ldap_schema_new(ctx);
-    schema->object_classes_capacity = 1;
+
```

Comment:
This test tests internal capacity of a schema structure now that we use `GHashTable` it can be safely deleted.